### ISSUE TITLE
Softmax classification

### ISSUE THREAD

**Issue by igardenia** [jwyang/faster-rcnn.pytorch#582]
Is the score value obtained by the softmax classification network not classified on the feature map? It can be used to select the anchor.
